### PR TITLE
[wip] fix 2263: Framework restriction is lost for global build folder

### PR DESCRIPTION
**Pull request by 0x53A**
ref https://github.com/fsprojects/Paket/issues/2263

### CODE REVIEW

File: src/Paket.Core/Files/ProjectFile.fs [hunk 1]
```diff
@@ -1079,7 +1079,7 @@ module ProjectFile =
 
             let importTargets = defaultArg installSettings.ImportTargets true
             
-            let allFrameworks = applyRestrictionsToTargets ((thirdOf3 kv.Value)) KnownTargetProfiles.AllProfiles
+            let allFrameworks = applyRestrictionsToTargets ((thirdOf3 kv.Value)) (KnownTargetProfiles.AllDotNetStandardProfiles @ KnownTargetProfiles.AllDotNetProfiles)
```
Comment (0x53A) [inline src/Paket.Core/Files/ProjectFile.fs]: PLEASE REVIEW: I think we don't care about runtimes when deciding whether to install a .targets / .props, is that correct?

Comment (0x53A) [inline src/Paket.Core/Files/ProjectFile.fs]: ref https://github.com/fsprojects/Paket/issues/2238

**Comment by 0x53A** [comment]
Test failure:

```
 	System.dll!System.Uri.CreateThis(string uri, bool dontEscape, System.UriKind uriKind) + 0x105 bytes	
 	System.dll!System.Uri.Uri(string uriString) + 0x16 bytes	
>	Paket.Core.dll!Paket.Utils.createRelativePath(string root, string path) Line 163 + 0x18 bytes	F#
 	Paket.Core.dll!Paket.ProjectFileModule.globalTargetsNodes@930-4.Invoke(string path) Line 930 + 0x10 bytes	F#

```

image

**Comment by 0x53A** [comment]
@forki was the baseline update ok? especially https://github.com/fsprojects/Paket/pull/2272/files#diff-c8216d6e9932fe764472cecc1af0914cR50

**Comment by forki** [comment]
I dunno ;-)
These whole targets file things are confusing

Am 24.04.2017 10:33 vorm. schrieb "Lukas Rieger" <notifications@github.com>:

**Comment by 0x53A** [comment]
that's why the PR was still tagged __[WIP]__ =)

**Pull request merged**